Posts/2021/02/Get-Services, PowerShell 7 VS Windows PowerShell 5.1

```diff
@@ -0,0 +1,150 @@
+Start                            Method
+Stop                             Method
+WaitForStatus                    Method
+BinaryPathName                 Property
+CanPauseAndContinue            Property
+CanShutdown                    Property
+CanStop                        Property
+Container                      Property
+DelayedAutoStart               Property
+DependentServices              Property
+Description                    Property
+DisplayName                    Property
+MachineName                    Property
+ServiceHandle                  Property
+ServiceName                    Property
+ServicesDependedOn             Property
+ServiceType                    Property
+Site                           Property
+StartType                      Property
+StartupType                    Property
+Status                         Property
+UserName                       Property
+ToString                   ScriptMethod
+```
+
+```powershell
+**Windows Powershell 5.1**
+Name                         MemberType
+----                         ----------
+Name                      AliasProperty
+RequiredServices          AliasProperty
+Disposed                          Event
+Close                            Method
+Continue                         Method
+CreateObjRef                     Method
+Dispose                          Method
+Equals                           Method
+ExecuteCommand                   Method
+GetHashCode                      Method
+GetLifetimeService               Method
+GetType                          Method
+InitializeLifetimeService        Method
+Pause                            Method
+Refresh                          Method
+Start                            Method
+Stop                             Method
+WaitForStatus                    Method
+CanPauseAndContinue            Property
+CanShutdown                    Property
+CanStop                        Property
+Container                      Property
+DependentServices              Property
+DisplayName                    Property
+MachineName                    Property
+ServiceHandle                  Property
+ServiceName                    Property
+ServicesDependedOn             Property
+ServiceType                    Property
+Site                           Property
+StartType                      Property
+Status                         Property
+ToString                   ScriptMethod
+```
+
+## PowerShell 7
+
+The big difference is in the Property, now in Powershell 7, it's possible to read some additional properties that were not available in Windows PowerShell 5.1, such as **UserName, BinaryPathName, StartType.** So let's see how to read the **Username**, **BinaryPathName,** and **StartType** using PowerShell 7
+
+```powershell
+PS 7> Get-Service workstation | select  Username,Starttype,BinaryPathName
+```
+
+The output is a clear, with all the required result with one-liner native command.
+
+```powershell
```

Comment:
style the output with three backticks then powershell-console, not just powershell